CODEOWNERS

```diff
@@ -1,2 +1,2 @@
-* @nobl9/SRE
+* @nobl9/sre-manual
```

Comment:
do we end up w/ 2 entries this time?

Reply:
derp. ill change this one to just have the one.  didn't realize that it already had the new one

Reply:
updated